### PR TITLE
feat: Add Page Metadatas with I18N - MEED-2081 - Meeds-io/MIPs#48 

### PR DESCRIPTION
**Pull request by MayTekayaa**
This change will add the metadata with i18n for each page in the dapp.

### CODE REVIEW

File: deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/about-us.html [hunk 1]
```diff
@@ -3,8 +3,8 @@
 <meta name="robots" content="index, follow">
 <link rel="canonical" href="https://www.meeds.io/${lang}/qui-somme-nous/">
 <meta property="og:locale" content="fr_FR">
-<link rel="alternate" href="https://www.meeds.io/?lang=fr/qui-somme-nous/" hreflang="fr"/>
-<link rel="alternate" href="https://www.meeds.io/about-us/" hreflang="en"/>
+<link rel="alternate" href="https://www.meeds.io/qui-somme-nous?lang=fr" hreflang="fr"/>
```
Comment (plamarque) [inline deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/about-us.html]: there is a typo here. it should be qui-sommes-nous

File: deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/legals.html [hunk 2]
```diff
@@ -3,8 +3,8 @@
 <meta name="robots" content="index, follow">
 <link rel="canonical" href="https://www.meeds.io/${lang}/legales/">
 <meta property="og:locale" content="fr_FR">
-<link rel="alternate" href="https://www.meeds.io/?lang=fr/legales/" hreflang="fr"/>
-<link rel="alternate" href="https://www.meeds.io/legals/" hreflang="en"/>
+<link rel="alternate" href="https://www.meeds.io/legales?lang=fr" hreflang="fr"/>
```
Comment (plamarque) [inline deeds-dapp-webapp/src/main/webapp/WEB-INF/metadata/legals.html]: should rather be /mentions-legales

**Review by boubaker — DISMISSED** [review]

**Review by boubaker — APPROVED** [review]

**Comment by exo-swf** [comment]
Your PR triggers too many exo-ci builds! Please finish your work and then, set your PR ready! Thank you

File: deeds-dapp-webapp/src/main/webapp/static/i18n/messages_fr.properties [hunk 3]
```diff
@@ -780,4 +780,56 @@ kudos.description=La reconnaissance entre pairs avec les kudos booste le moral d
...
+metadata.tour.twitterTitle=Hub communautaire Web3 pour votre entreprise - Meeds DAO
+metadata.whitepaper.pageTitle=Meeds DAO Work Metaverse Livre Blanc
+metadata.whitepaper.pageDescription=Découvrez le livre blanc de Meeds DAO Work Metaverse et explorez le WoM, un système de hubs d'engagement dans le web3 qui promeut la transparence et l'équité dans le travail par des incitations justes et une reconnaissance objective.
+metadata.whitepaper.imageAlt=Meeds DAO Work Metaverse Livre Blanc
+metadata.whitepaper.twitterTitle=Meeds DAO Work Metaverse Livre Blanc
+metadata.marketplace.pageTitle=Metaverse et place de marché NFT - Web3 Hub | Meeds DAO
+metadata.marketplace.pageDescription=Parcourez Meeds DAO Metaverse et sa place de marché NFT. Explorez les offres du Web3 Hub qui changeront l'avenir du travail.
+metadata.marketplace.imageAlt=Metaverse et place de marché NFT - Web3 Hub | Meeds DAO
+metadata.marketplace.twitterTitle=Metaverse et place de marché NFT - Web3 Hub | Meeds DAO
+metadata.tokenomics.pageTitle=Ɱeeds Tokenomics -Jeton Ɱeed | Meeds DAO
```
Comment (plamarque) [inline deeds-dapp-webapp/src/main/webapp/static/i18n/messages_fr.properties]:
```suggestion
metadata.tokenomics.pageTitle=Ɱeeds Tokenomics - Jeton Ɱeed | Meeds DAO
```

File: deeds-dapp-webapp/src/main/webapp/static/i18n/messages_fr.properties [hunk 4]
```diff
@@ -780,4 +780,56 @@ kudos.description=La reconnaissance entre pairs avec les kudos booste le moral d
...
+metadata.tokenomics.pageDescription=Découvrez le projet tokenomics de Meeds et tenez-vous au courant.
+metadata.tokenomics.imageAlt=Ɱeeds Tokenomics -Jeton Ɱeed | Meeds DAO
```
Comment (plamarque) [inline deeds-dapp-webapp/src/main/webapp/static/i18n/messages_fr.properties]:
```suggestion
metadata.tokenomics.imageAlt=Ɱeeds Tokenomics - Jeton Ɱeed | Meeds DAO
```

**Review by plamarque — DISMISSED** [review]
small typos

File: deeds-dapp-service/src/test/java/io/meeds/deeds/web/filter/RequestDispatcherFilterTest.java [hunk 5]
```diff
@@ -0,0 +1,50 @@
...
+    };
+    when(request.getRequestDispatcher(anyString())).thenReturn(dispatcher);
+  }
+
+  @Test
+  void testGetContentPage() throws Exception {
+    when(request.getRequestURL()).thenReturn(new StringBuffer("http://test/test"));
+    when(request.getServletPath()).thenReturn("/test");
+    when(request.getParameter("lang")).thenReturn("en");
+    dispatcherFilter.doFilter(request, response, chain);
```
Comment (boubaker) [inline deeds-dapp-service/src/test/java/io/meeds/deeds/web/filter/RequestDispatcherFilterTest.java:70]: there is not test after making the operation, please add assertions to make the test relevant

**Pull request merged**